pkgs/top-level/all-packages.nix

```diff
@@ -41731,4 +41733,6 @@ with pkgs;
   gitrs = callPackage ../tools/misc/gitrs { };
 
   wttrbar = callPackage ../applications/misc/wttrbar { };
+
+  ragnarwm = callPackage ../applications/window-managers/ragnarwm {};
```

Comment:
It's preferred to put the attribute… *somewhere* in this mess that isn't the very start/end of the file, to avoid conflicts.

Since this is inspired by it, maybe search for `dwm` and scroll down the list from there until you find a ~ alphabetically-correct spot for it. Between `qemacs` and `rime-cli` maybe?